tv/tv.cpp

```diff
@@ -162,6 +163,19 @@ llvm::cl::opt<unsigned> opt_tgt_unrolling_factor(
     llvm::cl::desc("Unrolling factor for tgt function (default=0)"),
     llvm::cl::cat(TVOptions), llvm::cl::init(0));
 
+#ifdef CLANG_PLUGIN
+llvm::cl::opt<bool> parallel_tv(
+    "tv-parallel",
+    llvm::cl::desc("Distribute TV load across cores (requires GNU Make, default=false)"),
+    llvm::cl::cat(TVOptions), llvm::cl::init(false));
+llvm::cl::opt<int> max_subprocesses(
+    "tv-max-subprocesses",
+    llvm::cl::desc("Maximum children this clang instance will have at any time (default=128)"),
+    llvm::cl::cat(TVOptions), llvm::cl::init(128));
+#else
+bool parallel_tv = false;
```

Comment:
I think this can be enabled for both opt & clang plugins

Reply:
got it.

I think it might as well be turned on by default since none of the parallel stuff will work at all unless the jobserver is detected, but we can switch this later if we want.